charts/frontend/Chart.yaml

```diff
@@ -32,3 +32,7 @@ dependencies:
 - name: silta-release
   version: ^1.x
   repository: file://../silta-release
+- name: redis
+  version: 18.19.x
```

Comment:
The latest seems to be 19.1.5. Could you please test with that? :)

Reply:
Updated in 16947a7a6b52b4d054cce26efa519eff49711250